python_modules/libraries/dagster-dbt/dagster_dbt/core/resource.py

```diff
@@ -588,10 +590,6 @@ def my_dbt_op(dbt: DbtCliResource):
         with suppress(DagsterInvalidPropertyError):
             assets_def = context.assets_def if context else None
 
-        context = (
```

Comment:
this was added so that the functions that are called later in the function only need to support `OpExecutionContext`. I updated those functions to support `AssetExecutionContext` too, so this can be removed